develop/tutorials/articles/xx-item-selector/creating-custom-entites-for-the-item-selector.markdown

```diff
@@ -40,7 +40,15 @@ Follow the steps below to create your custom Item Selector criterion:
         }
         
     This class acts as a flag of sorts, allowing the Item Selector to accept it 
-    as a valid criterion. That's all that is needed for this class.
+    as a valid criterion.
+
+    It could be used to pass information to the view if
+    needed. For example, if you need to show the images attached to a certain
+    entity you could add a field (adn setters and getters ) with the entity id
+    in order to use it in the view.
+
+    You should be aware that the criterion fields should be serializable and
+    should expose a public empty constructor due framework internals need it.
 
```

Comment:
That the framework requires criterion implementations fields to be serializable. Developers could create new constructors with this fields, but is mandatory (because  framework requires it) a public empty constructor